src/ducks/account/helpers.js

```diff
@@ -46,3 +46,71 @@ export const distanceInWords = distance => {
 
   return 'Balance.updated_at.n_days_ago'
 }
+
+/**
+ * Get the translated type of an account
+ * @param {Object} account
+ * @param {Function} translate
+ * @returns {String}
+ */
+export const getAccountTranslatedType = (account, translate) => {
+  const accountTypesMap = {
```

Comment:
This const can be extract to use it on `getAccountTranslatedType` and `getAccountType` ?

Reply:
Actually I forgot to remove `getAccountTranslatedType`